botx/models/chats.py

```diff
@@ -28,3 +29,47 @@ class ChatFromSearch(BotXBaseModel):
 
     #: users in chat.
     members: List[UserFromChatSearch]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+
+class BotChat(BotXBaseModel):
```

Comment:
`ChatFromList` would be better

Reply:
From what list? This name is my suggestion, I think it's ok.

Reply:
Chat can has different signature. That was reason to name `ChatFromInfo`

Reply:
I think @Opti213 is right, there may be more different Chat classes